### ISSUE TITLE
blur does not hide the menu

### ISSUE THREAD

**Issue by isilmartinez** [thebritican/elm-autocomplete#54]
click on first input box  and type, click on second input box and type
Both overlays remain opened.

Version: 4.0.3
Elm 0.18

**Comment by ericgj** [comment]
See #33 .  But not sure the status of this library, it may be in the process of being moved/rewritten in https://github.com/thebritican/elm-menu

**Comment by isilmartinez** [comment]
Yeah that could do the work (I was doing it with the standard onBlur event and applying hidden when there were no focus) but the onBlur   applies before the onClick, I have seen that onMouseDown were present in older versions but looks like it was removed
